.github/workflows/publish.yml

```diff
@@ -13,17 +15,70 @@ jobs:
 
       - name: Set up Quarto
         uses: quarto-dev/quarto-actions/setup@v2
+
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          pak-version: devel
+
+      - name: Render Quarto Project
+        uses: quarto-dev/quarto-actions/render@v2
+
+      - name: Restore lychee cache
+        uses: actions/cache@v3
         with:
-          # To install LaTeX to build PDF book
-          tinytex: true
-          # uncomment below and fill to pin a version
-          # version: SPECIFIC-QUARTO-VERSION-HERE
+          path: .lycheecache
+          key: cache-lychee-${{ github.sha }}
+          restore-keys: cache-lychee-
 
-      # add software dependencies here
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1
+        with:
+          fail: true
+          args: "docs --exclude-loopback --insecure --exclude-mail --exclude-path docs/site_libs --cache --max-cache-age 1d"
 
-      - name: Publish to GitHub Pages (and render)
-        uses: quarto-dev/quarto-actions/publish@v2
+      - name: Read lychee's out.md
+        if: failure()
+        id: lychee-output
+        uses: juliangruber/read-file-action@v1
         with:
-          target: gh-pages
+          path: ./lychee/out.md
+
+      - uses: mshick/add-pr-comment@v2
+        if: failure()
+        with:
+          message: ${{ steps.lychee-output.outputs.content }}
+
+      - name: Save lychee cache
+        uses: actions/cache/save@v3
+        if: always()
+        with:
+          path: .lycheecache
+          key: ${{ steps.restore-cache.outputs.cache-primary-key }}
+
+      - uses: actions/setup-node@v3
+        timeout-minutes: 2
+        with:
+          node-version: 16
+
+      - name: html5 validate
```

Comment:
see #26 
This is a blocker

Reply:
I don't think the html validation should be a blocker? If quarto doesn't pass it, let's skip the html validation.